.github/workflows/Test.yml

```diff
@@ -69,14 +69,16 @@ jobs:
             matrix:
                 os: [ 'ubuntu-latest' ]
                 typo3: [ '^11.5.26', '^12.4', '^13.4' ]
-                php: [ 8.1, 8.2, 8.3 ]
+                php: [ 8.1, 8.2, 8.3, 8.4 ]
                 dependency-version: [ lowest, stable ]
                 exclude:
+                    - typo3: "^11.5.26"
+                      php: 8.4
                     - typo3: "^13.4"
                       php: 8.1
                 include:
                     -   os: 'windows-latest'
-                        php: 8.3
+                        php: 8.4
```

Comment:
@helhum Can you adjust the required checks accordingly or relax them for a short time? Otherwise the PR cannot be merged with this change.

Reply:
@mbrodala I will adjust and merge this one later today. 

Thanks again @eliashaeussler and @mbrodala !